docs/mkdocs/docs/index.md

```diff
@@ -323,6 +323,6 @@ Note the timestamps - we've read the data prior to the _append_ operation. Pleas
 
 !!! note "Versioning & Prune Previous"
 
-    By default, `write`, `append`, and `update` operations will remove the previous versions to save on space. 
-
-    Use the `prune_previous` argument to control this behaviour. 
+    By default, `write`, `append`, and `update` operations will **not** remove the previous versions. Please be aware that this will consume more space.
```

Comment:
Please place each sentence on its own line.

Reply:
I think they are. "Please be aware that this will consume more space" is related to previous sentence

Reply:
`Please be aware that this will consume more space.` is on the same line. Line 326.

It does make it marginally harder to comment.

Reply:
yes, I will leave it as is then